vital/types/object_track_set.cxx

```diff
@@ -68,5 +68,28 @@ ::object_track_set( std::vector< track_sptr > const& tracks )
 {
 }
 
+/// Clone the track state (polymorphic copy constructor)
+track_state_sptr
+object_track_state::clone ( clone_type ct ) const
+{
+  if ( ct == clone_type::DEEP )
+  {
+    auto new_detection =
+      ( this->detection_ ? this->detection_->clone() : nullptr );
+
+    auto copy = std::make_shared< object_track_state >(
+      this->frame(), this->time(), std::move( new_detection ) );
+
+    copy->set_image_point(this->image_point_);
```

Comment:
Über-pedantic: [[ws.space]](../tree/master/doc/code-style.rst) :wink:. (This is probably the least enforced and most inconsistent, but I do notice that these lines are inconsistent with the surrounding code...)